helm/repositories.bzl

```diff
@@ -124,12 +125,9 @@ exports_files(["helm{ext}"])
         ext = ext,
     ))
 
-    platform = "{}_{}".format(repository_ctx.os.name, repository_ctx.os.arch)
-    platform_repo_name = _helm_bin_repo_name(platform)
-
-    repository_ctx.symlink("../{platform_repo_name}/helm{ext}".format(
+    repository_ctx.symlink("../{name}_{platform}/helm{ext}".format(
         name = repository_ctx.attr.name,
-        platform_repo_name = platform_repo_name,
+        platform = repo_utils.platform(repository_ctx),
```

Comment:
This is a nice util from `aspect_bazel_lib`. I can vendor the implementation if you don't want to add an explicit dependency on it. I realize there is a dependency currently, but that's due to the rules_oci bug.

Reply:
I would much prefer to keep the set of dependencies minimal if possible 🙏

Reply:
https://github.com/abrisco/rules_helm/pull/79#discussion_r1597540398

Reply:
Vendored under 3rdparty.